configure.acr

```diff
@@ -1,6 +1,6 @@
 PKGNAME radare2
 VERSION 2.8.0-git
-CONTACT pancake ; pancake@nopcode.org
+CONTACT radare2 ; https://github.com/radare/radare2/issues
```

Comment:
this is suposed to be an email address, not an url. and this pr is probably not the place to discuss and change this